<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smart Oil Press Management - Clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow { background:#F8EDE3; }
    QPushButton {
      background:#A2B29F;
      border-radius:6px;
      padding:6px;
      font-weight:bold;
      color:#2E2E2E;
    }
    QPushButton:hover { background:#798777; color:white; }
    QGroupBox {
      background:#BDD2B6;
      border-radius:8px;
      margin-top:8px;
    }
    QGroupBox::title {
      subcontrol-origin: margin;
      left:10px;
      padding:0 5px;
      font-weight:bold;
    }
    QTabBar::tab {
      background:#C8E6C9;
      padding:8px;
      border-top-left-radius:6px;
      border-top-right-radius:6px;
      color:#2E2E2E;
    }
    QTabBar::tab:selected {
      background:#A5D6A7;
      color:#1B5E20;
      font-weight:bold;
    }
    QTabWidget::pane {
      background:white;
      border: 1px solid #A5D6A7;
      border-radius:4px;
    }
    QTableWidget {
      background-color:white;
      border: 1px solid #E0E0E0;
      gridline-color:#E0E0E0;
    }
    QTextEdit {
      background-color:white;
      border: 1px solid #E0E0E0;
      border-radius:4px;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="sidebarFrame">
      <property name="maximumWidth">
       <number>200</number>
      </property>
      <property name="styleSheet">
       <string notr="true">background:#A2B29F;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblLogo">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>:/logo_oil_press.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEmploye">
         <property name="text">
          <string>EMPLOYÉ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEquipe">
         <property name="text">
          <string>ÉQUIPE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClient">
         <property name="text">
          <string>CLIENT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLaboratoire">
         <property name="text">
          <string>LABORATOIRE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCommande">
         <property name="text">
          <string>COMMANDE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnMachine">
         <property name="text">
          <string>MACHINE</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="mainContentLayout">
      <item>
       <layout class="QHBoxLayout" name="contentLayout">
        <item>
         <widget class="QGroupBox" name="grpRechercheFiltre">
          <property name="title">
           <string>Recherche et Filtres</string>
          </property>
          <property name="maximumWidth">
           <number>250</number>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="lblNom">
             <property name="text">
              <string>Nom</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="txtRechercheNom"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lblType">
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cmbType"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lblStatut">
             <property name="text">
              <string>Statut</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="cmbStatut"/>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="btnAppliquerFiltre">
             <property name="text">
              <string>Appliquer</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="styleSheet">
           <string notr="true">QTabWidget::pane { background: white; border: 1px solid #A5D6A7; }</string>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::North</enum>
          </property>
          <property name="usesScrollButtons">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="tabListeClients">
           <attribute name="title">
            <string>Liste des Clients</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableWidget" name="tableClients">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QTableWidget { background-color: white; }</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabStatistiques">
           <attribute name="title">
            <string>Statistiques</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTextEdit" name="txtStatistiques">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabAlertes">
           <attribute name="title">
            <string>Alertes</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QTextEdit" name="txtAlertes">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="rightPanelLayout">
          <item>
           <widget class="QGroupBox" name="grpActionsRapides">
            <property name="title">
             <string>Actions Rapides</string>
            </property>
            <property name="maximumWidth">
             <number>180</number>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="btnActionAjouter">
               <property name="text">
                <string>➕ Ajouter</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnActionModifier">
               <property name="text">
                <string>✏️ Modifier</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnActionSupprimer">
               <property name="text">
                <string>❌ Supprimer</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="grpInfosClient">
            <property name="title">
             <string>Infos Client</string>
            </property>
            <property name="maximumWidth">
             <number>180</number>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="lblInfoStatut">
               <property name="text">
                <string>Statut : -</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblInfoScore">
               <property name="text">
                <string>Score : -</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblInfoAlertes">
               <property name="text">
                <string>Alertes : -</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
